src/datachain/query/dispatch.py

```diff
@@ -44,44 +53,6 @@ def get_n_workers_from_arg(n_workers: Optional[int] = None) -> int:
     return n_workers
 
 
-# For more context on the get_from_queue and put_into_queue functions, see the
-# discussion here:
-# https://github.com/iterative/dvcx/pull/1297#issuecomment-2026308773
-# This problem is not exactly described by, but is also related to these Python issues:
-# https://github.com/python/cpython/issues/66587
-# https://github.com/python/cpython/issues/88628
-# https://github.com/python/cpython/issues/108645
-
-
-def get_from_queue(queue: Queue) -> Any:
```

Comment:
are we moving things around in this case? can it be done please as a separate PR if that's the case? (PR is large, quite nuanced any reduction in changes can help to review it).

Reply:
> are we moving things around in this case?

We do.

`dispatch.py` grow up quite a big after my initial updates and I was thinking about the way to reduce it's size and move out some parts to separate file (`queue.py`).

I can do it is separate PR though 👀